packages/lumx-react/src/components/autocomplete/AutocompleteMultiple.stories.tsx

```diff
@@ -149,6 +149,7 @@ export const simple = ({ theme }: any) => {
             onBlur={onBlur}
             selectedChipRender={renderChip}
             icon={mdiFlag}
+            label="toto"
```

Comment:
Maybe use a better label?

Reply:
You don't say... ^^

Nice catch though